presto-main/src/main/java/com/facebook/presto/operator/scalar/ColorFunctions.java

```diff
@@ -149,8 +153,10 @@ public static long color(@SqlType(StandardTypes.DOUBLE) double fraction, @SqlTyp
     }
 
     @ScalarFunction
-    @SqlType(StandardTypes.VARCHAR)
-    public static Slice render(@SqlType(StandardTypes.VARCHAR) Slice value, @SqlType(ColorType.NAME) long color)
+    @LiteralParameters({"x", "y"})
+    @Constraint(variable = "y", expression = "x + 15")
```

Comment:
Done.
Note that this and many other return types are strictly connected to the implementation and some external factor (eg. in this example - how long are color escape codes for shell). This is something I don't like but I don't think we can avoid this in current (relatively simple) implementation.